src/utils/utils.tsx

```diff
@@ -25,29 +27,34 @@ export const getGeography = (city: string[] | null | undefined, state: string[]
     return Translate("Not Reported");
   }
 
-  function renderOutGeography(geo: string[] | null | undefined) {
+  function renderOutGeography(geo: string[] | string | null | undefined) {
     if (!geo || geo.length === 0) {
-      return undefined
+      return ""
+    }
+    if (typeof geo === "string"){
+      return geo.trim() + ", "
     }
-    else if (geo.length > 1 && typeof geo !== "string") {
-      return geo?.join(", ");
+    else {
+      return geo.join(", ") + ", ";

```

Comment:
I think we should stay consistent with either join or the `+` operator for string concatenation!

Reply:
here the join concatenates the words in the list geo, and the + adds a comma to the end (idk if I can add the comma to the end with join?)

Reply:
"If the array has only one item, then that item will be returned without using the separator."
Also "property join does not exist on type string"

Reply:
yeah the logic here is fine imo, resolving